src/components/atoms/Button/button.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { Button } from "react-bootstrap"
+
+const Button = (props) => {
```

Comment:
just naming preferences..

if you already import Button from library like bootstrap or antd or anything,,

I think it will be better if you named it AtomButton or something to make sure it is distinct from other component imported @alramdein